gradlew

```diff
@@ -130,6 +130,8 @@ fi
 if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
     APP_HOME=`cygpath --path --mixed "$APP_HOME"`
     CLASSPATH=`cygpath --path --mixed "$CLASSPATH"`
+    
+    JAVACMD=`cygpath --unix "$JAVACMD"`
 
```

Comment:
Something funny going on here... I'm assuming this is a mistake from resolving the merge conflicts? Same with the gradle-wrapper.jar changes?

Reply:
Interesting... didn't see that one. I'll have a look at it. Thanks again @ptuomola

Reply:
@ptuomola this seems to be still failing... how about kicking this one in the trash can and starting from scratch? I think that would save us a ton of time... going to do that.